package.json

```diff
@@ -1,53 +1,67 @@
 {
   "name": "eddie",
-  "version": "1.0.0",
+  "version": "0.1.0",
+  "private": true,
   "description": "## What is our community about - We believe open source is for everyone. - Our community is inclusive and a safe space for all. - Please welcome new people. - The best way to learn is to help someone else. - [Code Of Conduct](https://github.com/EddieJaoudeCommunity/EddieJaoudeCommunity.github.io/blob/main/CODE_OF_CONDUCT.md)",
-  "main": "index.js",
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": false
   },
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "@types/jest": "^24.0.0",
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "cypress": "^5.4.0",
+    "cypress-cucumber-preprocessor": "^4.0.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-prettier": "^3.1.4",
+    "husky": "^4.3.0",
+    "lint-staged": "^10.4.2",
+    "prettier": "^2.1.2",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-scripts": "3.4.4",
+    "typescript": "~3.7.2"
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
+      "prettier --write"
+    ]
+  },
   "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test:react": "react-scripts test",
     "test": "cypress run",
-    "test:run": "concurrently -k --success first \"npm run test\" \"npm run start\"",
-    "lint": "eslint ./src ./cypress --ext .svelte,.js",
-    "lint:fix": "eslint ./src ./cypress --ext .svelte,.js --fix",
-    "setup": "npm install && npm run lint && npm run build && npm run test:run",
     "cypress:open": "cypress open",
-    "build": "rollup -c",
-    "dev": "rollup -c -w",
-    "start": "sirv public --port 8080"
+    "eject": "react-scripts eject"
```

Comment:
we should have the same scripts as the `develop` branch. Right now on the PR we have a Quickstart section on the `README.md`, but when running `npm run setup` it fails because the script was deleted.
Please add these scripts back, but change what gets executed according to React. For example, your `start` and `build` script are fine 👍.